omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/TaskDefinitionResource1_8.java

```diff
@@ -0,0 +1,184 @@
+			description.addLink("ref", ".?v=" + RestConstants.REPRESENTATION_REF);
+			description.addLink("full", ".?v=" + RestConstants.REPRESENTATION_FULL);
+			description.addSelfLink();
+			return description;
+		} else if (rep instanceof FullRepresentation) {
+			DelegatingResourceDescription description = new DelegatingResourceDescription();
+			description.addProperty("uuid");
+			description.addProperty("name");
+			description.addProperty("description");
+			description.addProperty("taskClass");
+			description.addProperty("startTime");
+			description.addProperty("lastExecutionTime");
+			description.addProperty("repeatInterval");
+			description.addProperty("startOnStartup");
+			description.addProperty("startTimePattern");
+			description.addProperty("started");
+			description.addProperty("properties");
+			description.addSelfLink();
+			description.addLink("ref", ".?v=" + RestConstants.REPRESENTATION_REF);
+			return description;
+		} else if (rep instanceof RefRepresentation) {
+			DelegatingResourceDescription description = new DelegatingResourceDescription();
+			description.addProperty("uuid");
+			description.addProperty("name");
+			description.addProperty("description");
+			description.addProperty("taskClass");
+			description.addSelfLink();
+			return description;
+		}
+		return null;
+	}
+	
+	/**
+	 * @see org.openmrs.module.webservices.rest.web.resource.impl.BaseDelegatingResource#getCreatableProperties()
+	 */
+	@Override
+	public DelegatingResourceDescription getCreatableProperties() {
+		DelegatingResourceDescription description = new DelegatingResourceDescription();
+		description.addRequiredProperty("name");
+		description.addRequiredProperty("description");
+		description.addRequiredProperty("taskClass");
+		description.addRequiredProperty("startTime");
+		description.addRequiredProperty("repeatInterval");
+		description.addRequiredProperty("startOnStartup");
+		description.addRequiredProperty("properties");
+		return description;
+	}
+	
+	//	/**
+	//	 * @see org.openmrs.module.webservices.rest.web.resource.impl.BaseDelegatingResource#getUpdatableProperties()
+	//	 */
+	//	@Override
+	//	public DelegatingResourceDescription getUpdatableProperties() {
+	//		DelegatingResourceDescription description = new DelegatingResourceDescription();
+	//		description.addRequiredProperty("name");
+	//		description.addRequiredProperty("description");
+	//		description.addRequiredProperty("taskClass");
+	//		description.addRequiredProperty("startTime");
+	//		description.addRequiredProperty("repeatInterval");
+	//		description.addRequiredProperty("startOnStartup");
+	//		description.addRequiredProperty("properties");
+	//		return description;
+	//	}
+	//
+	/**
+	 * @see org.openmrs.module.webservices.rest.web.resource.impl.DelegatingCrudResource#save(java.lang.Object)
+	 */
+	@Override
+	public TaskDefinition save(TaskDefinition taskDefinition) {
+		//Context.getSchedulerService().saveTaskDefinition(taskDefinition);
```

Comment:
What is the above line for? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotcommentoutcode